default/generated/technology-usage/22-javaee-technology-usage.windup.yaml

```diff
@@ -10,9 +11,11 @@
       filepaths:
       - batch.xml
       namespaces:
-        "": http://xmlns.jcp.org/xml/ns/javaee
+        ? ''
```

Comment:
I am worried about this change, do we have a test to make sure this still works?

Reply:
Good point, going to look on this.